src/pyspiffe/config.py

```diff
@@ -0,0 +1,90 @@
+import os
+import ipaddress
+from urllib.parse import ParseResult, urlparse
+from typing import Dict, List, Tuple
+
+
+SPIFFE_ENDPOINT_SOCKET = 'SPIFFE_ENDPOINT_SOCKET'
+
+
+class ConfigSetter:
+    def __init__(self):
+        self._apply_default_config()
+        self._apply_environment_variables()
+
+    def get_config(self) -> Dict:
+        return self._config
+
+    def _apply_default_config(self):
+        self._config = {
+            SPIFFE_ENDPOINT_SOCKET: None,
+        }
+
+    def _apply_environment_variables(self):
+        endpoint_socket = os.environ.get(SPIFFE_ENDPOINT_SOCKET)
+
+        if endpoint_socket is not None:
```

Comment:
nit: `endpoint_socket` is of type `str` and strings are falsy (None and empty strings evaluates to False) in python so this can be reduced to:
```
if endpoint_socket:
    <code>
```

Reply:
thanks!, good tip 👌